web/docs/getting-started.md

```diff
@@ -121,6 +121,23 @@ wasp start # Serves the web app.
 
 That's it :tada:! You have successfully created and served a new web app at <http://localhost:3000> and Wasp is serving both frontend and backend for you.
 
+<!-- <div style={{backgroundColor: 'rgba(84, 199, 236, 0.15)'}}> -->
+
+:::info For Visual Studio Code
+
+If you are using VSCode, install our [Wasp language extention](https://marketplace.visualstudio.com/items?itemName=wasp-lang.wasp). 
+
+The extension has the following features tailored to meet your development needs:
+
+* Syntax highlighting for .wasp files
+* Snippets for .wasp files
+* Wasp language server
+* live reporting of compilation errors
+* autocompletion
+
+:::
+<!-- </div> -->
+
```

Comment:
Looks good, please include a screenshot with your PR (for documentation and reviewing purposes :)). I can test it locally tomorrow.

Reply:
![image](https://user-images.githubusercontent.com/55356373/196415457-1e28f7d1-5bfd-468d-b546-a3fdc73ec4a3.png)

Reply:
Looks nice! I'm handing this over to @Martinsos since he had most of the comments :)

He'll give you the final approaval.

Reply:
Thanks, Sodic. I mentioned you in issue #756. Kindly check it out